lib/src/config.dart

```diff
@@ -176,20 +200,23 @@ void _parseDeletionPatterns(yaml, GlobalConfig config) {
       .addAll((patterns as YamlList).map((pattern) => new RegExp(pattern)));
 }
 
-void _parseFilesToLoad(yaml, GlobalConfig config) {
+Future _parseFilesToLoad(yaml, GlobalConfig config) async {
   var filePaths = _parseStringList(yaml, 'files_to_load');
   if (filePaths == null) return;
-  config.filesToLoad.addAll(filePaths.map((filePath) {
+  config.filesToLoad.addAll(await Future.wait(filePaths.map((filePath) async {
     var parts = filePath.split(':');
     if (parts.length == 1) {
       return platformIndependentPath(parts[0]);
     } else if (parts.length == 2 && parts[0] == 'package') {
-      return path.join('packages', platformIndependentPath(parts[1]));
+      Uri uri = await config.packageResolver.resolveUri(filePath);
+      String p = platformIndependentPath(uri.toFilePath());
+      p = p.replaceAll("%", r'%25'); // Reset url encoding
```

Comment:
ok, forget my previous comment. I had to undo the change to understand it.

I think the reason is that when the path is passed to the `analyzer` node script, it will get unescaped again by node parser . 

This could be done [here](https://github.com/dam0vm3nt/custom-element-apigen/blob/pkg_compat/lib/generate_dart_api.dart#L159) just before passing it to the script actually.